### PR TITLE
C++ Pairs STL

### PR DESCRIPTION
**Pull request by Mrjoy832**
## Related Issue
- #382 

## Proposed Changes
1. Added Code, output of STL of pairs in **LearnCPP/S-STL/C-Containers/P-Pairs**
2. Added readme file describing all about pairs and how to implement.

## Checklist

- [X] ✅ My code follows the code style of this project.
- [X] 📝 My change requires a change to the README.
- [X] 🎀 I have updated the README accordingly.
- [X] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [X] ✨ I have added tests to cover my changes.
- [X] 🚩 All new and existing tests passed.
- [X] 🌟 ed the repo

> STL is helped to implement many hard data structures easily by defining or implementing some in built functions, may containers are there, Pair is one among them, which takes two values into it. It has lots of inbuilt functions also to implement 
swiping, copying, and helps the problems which need two values as a pair.
**So I have added well-defined Code, description, and output also, that all are needed to start coding and implementing it**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Output Screenshots
| Screenshot #1 
output

### CODE REVIEW

**Comment by Mrjoy832** [comment]
I will be upadting the readme soon

**Comment by Mrjoy832** [comment]
> I will be upadting the readme soon

Updated the readme and all is set..please check

**Comment by Lakhankumawat** [comment]
## Uncheck the checks which do not apply to you !!

**Comment by Mrjoy832** [comment]
> ## Uncheck the checks which do not apply to you !!

Okay

**Comment by siddhi-244** [comment]
I see 3 files are committed!

**Comment by Mrjoy832** [comment]
@siddhi-244, Hey every time for a new issue when I create a new branch and add a file and try to commit..after committing why all my previous files are also committing ...that's why 3 files are committed...whats the issue please help me.

**Comment by Mrjoy832** [comment]
@siddhi-244, updated files please check

**Comment by siddhi-244** [comment]
Hey, just check the tasks which you have completed

**Comment by Mrjoy832** [comment]
> Hey, just check the tasks which you have completed 

@siddhi-244 , done

**Review by siddhi-244 — APPROVED** [review]
LGTM

File: S-STL/README.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+> # Pairs in C++ STL
```
Comment (Lakhankumawat) [inline S-STL/README.md:1]: add your README inside P-Pairs Folder

Comment (Mrjoy832) [reply S-STL/README.md:1]: Okay

**Comment by Mrjoy832** [comment]
@Lakhankumawat , Updated as your mentions

**Review by Lakhankumawat — APPROVED** [review]
LGTM 🌟

**Comment by Lakhankumawat** [comment]
## Resolve the conflicts

**Comment by Mrjoy832** [comment]
> ## Resolve the conflicts 

I solved all

**Comment by Lakhankumawat** [comment]
image

I am unable to resolve the conflits from my side so i would recommend you to create a new PR and DM Me your PR no in discord

Make sure you have same changes in PR like this one ,i'll merge it .

**Comment by Lakhankumawat** [comment]
Closing ...... unresolved conflicts ...

**Comment by Mrjoy832** [comment]
> Closing ...... unresolved conflicts ...

because late reviews from your side my issue was closed .... so that its showing

**Comment by Mrjoy832** [comment]
@Lakhankumawat okay

**Pull request closed without merging**